packages/terra-tabs/src/common-tabs/CommonTabs.jsx

```diff
@@ -50,6 +50,19 @@ const propTypes = {
    * Parameters: 1. Event 2. Selected pane's key
    */
   onChange: PropTypes.func,
+  /**
+   * Callback function when add button selection has changed.
+   * Parameters: 1. Event 2. Selected pane's key
+   */
+  onSelectAddButton: PropTypes.func,
+  /**
+   * The label to set on the add icon element.
+   */
+  ariaLabelAddTab: PropTypes.string,
+  /**
+   * Callback function when a tab is closing. It receives three parameters.
```

Comment:
can we mention what are those three parameters here !
and description should be more specific `Callback function triggered on close button click.`

Reply:
added the description.